react-backend/routes/cbauthentication/checkCBlogin.js

```diff
@@ -24,28 +24,31 @@ router.post('/', (req, res, next) => {
       );
     } else if (validator.isEmail(data.formEmail)) {
       data.formPswd = hashCB(data.formPswd);
-      getCBlogindetailsvalid(data.formEmail, data.formPswd, (error, result) => {
-        if (error) {
+      getCBlogindetailsvalid(data.formEmail, data.formPswd)
+        .then((result) => {
+          if (result) {
+            // if CB exists we want to create jwt and send it to the frontend
+            const token = jwt.sign({ email: data.formEmail }, process.env.SECRET);
+            res.send(
+              JSON.stringify({
```

Comment:
I think this was probably already like this but I'm pretty sure you don't need to stringify this, express should take care of that.